.github/workflows/report-release-vulnerabilities.yaml

```diff
@@ -22,12 +24,17 @@ jobs:
         run: curl --location --silent "https://github.com/google/go-containerregistry/releases/download/$(curl -s https://api.github.com/repos/google/go-containerregistry/releases/latest | jq -r '.tag_name')/go-containerregistry_$(uname -s)_$(uname -m | sed -e 's/aarch64/arm64/').tar.gz" | sudo tar -xzf - -C /usr/local/bin crane
       - name: Install dyff
         run: curl --silent --location https://raw.githubusercontent.com/homeport/dyff/main/scripts/download-latest.sh | bash
+      - name: Install Ko
+        uses: ko-build/setup-ko@v0.7
+        with:
+          version: v0.17.1
```

Comment:
Docs: [Select ko version to install](https://github.com/ko-build/setup-ko/tree/main?tab=readme-ov-file#select-ko-version-to-install)
> By default, ko-build/setup-ko installs the [latest released version of ko](https://github.com/ko-build/ko/releases).

Wouldn't that be nice actually?

Reply:
Yep, let's do that for all actions in a separate PR.